scripts/fedora/openstack.sh

```diff
@@ -9,7 +9,6 @@ dracut -f
 
 if [ $(uname -m)=="ppc64" -o $(uname -m)=="ppc64le" ]
   then
-    yum -y update
```

Comment:
yum redirects to dnf anyway, and I removed it because it's handled elsewhere and was doing nothing for ppc64-diag installation.